### PR TITLE
Support powermails EM setting replaceIrreWithElementBrowser

### PR DESCRIPTION
**Pull request by christophlehmann**
and show only form related fields and fieldsets in conditions

### CODE REVIEW

File: Classes/UserFunc/GetPowermailFields.php [hunk 1]
```diff
@@ -280,4 +285,12 @@ public function getDefaultFieldTypesForQuery()
...
         return ArrayUtility::getQuotedList($fieldTypes);
     }
+
+    protected function getFormUidFromAjaxRequest() {
+        if (isset(GeneralUtility::_GP('ajax')[0])) {
+            preg_match('/data-[0-9]*-' . ConditionContainer::TABLE_NAME . '-([0-9]*)/', GeneralUtility::_GP('ajax')[0], $matches);
+            $conditionContainer = (int)$matches[1];
+            return $this->getFormUidFromConditionContainer($conditionContainer);
+        }
+    }
```
Comment (christophlehmann) [inline Classes/UserFunc/GetPowermailFields.php]: Possibly not needed in 9.5+ when https://review.typo3.org/c/Packages/TYPO3.CMS/+/61480 is merged

**Comment by christophlehmann** [comment]
Hey Alex,

is there a chance to get this merged? :-)

**Comment by einpraegsam** [comment]
No, a merge is not planned. I'm thinking of removing the feature replaceIrreWithElementBrowser
 in powermail - sorry.

**Comment by christophlehmann** [comment]
Okay. Whats wrong with replaceIrreWithElementBrowser?

**Comment by einpraegsam** [comment]
This is just a test feature that we never fully supported

**Pull request closed without merging**